specification/cognitiveservices/data-plane/Speech/VideoDubbing/preview/2024-05-20-preview/video-dubbing.json

```diff
@@ -0,0 +1,849 @@
+                "type": "string",
+                "format": "uri"
+              }
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a translation": {
+            "$ref": "./examples/delete_translation.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Translation"
+        ],
+        "description": "Creates a new translation.",
+        "operationId": "Translations_Create",
+        "translation": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/PathResourceId"
+          },
+          {
+            "$ref": "#/parameters/QueryApiVersion"
+          },
+          {
+            "in": "body",
+            "name": "translation",
+            "description": "translation definition",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Translation"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/Translation"
+            },
+            "headers": {
+              "Operation-Id": {
+                "type": "string",
+                "description": "ID of the operation."
+              },
+              "Operation-Location": {
+                "description": "The location of the operation job. Use this URL to monitor operation status.",
+                "type": "string",
+                "format": "uri"
+              },
+              "retry-after": {
+                "description": "Indicate the minimum number of seconds the client should wait before polling (GETing) the status monitor URL again for an update.",
+                "type": "string",
+                "format": "uri"
+              }
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a project": {
+            "$ref": "./examples/create_translation.json"
+          }
+        }
+      }
+    },
+    "/translations/{id}:edit": {
```

Comment:
Why do you introduce a new API. In your original design, it's together with translation API.

Reply:
For content editing is defined as an LRO operation.